cypress/support/kaoto-ui-commands/steps.js

```diff
@@ -80,6 +80,7 @@ Cypress.Commands.add('openStepConfigurationTab', (step, EIP, stepIndex) => {
     if (!EIP) {
         cy.get('[data-testid="kaoto-right-drawer"]').should('be.visible');
     }
+    cy.waitVisualizationUpdate();
```

Comment:
This should be workarround for https://github.com/KaotoIO/kaoto-ui/issues/1855

Reply:
Awesome, I'll link your message on that issue, right now is just saying that the issue got mentioned on this pull request and I think it will come in handy to ensure that the bug was fixed.